.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+
+## Contributing to Issues
+
+* Check if the issue you are going to file already exists in our [GitHub issues][open-issue].
+* If you can't find your issue already,
+  [open a new issue](https://github.com/PowerShell/PSReadLine/issues/new/choose),
+  making sure to follow the directions as best you can.
+* If the issue is marked as [`Up-for-Grabs`][up-for-grabs],
+  the PSReadLine Maintainers are looking for help with the issue.
+
+## Contributing to Documentation
+
+The documentation is located in the [docs][psreadline-docs] folder.
+The markdown files there are converted to the PowerShell `help.xml` file via [platyPS][platy-ps] during the build.
+
+## Contributing to Code
+
+### Forks and Pull Requests
+
+GitHub fosters collaboration through the notion of [pull requests][using-prs].
+On GitHub, anyone can [fork][fork-a-repo] an existing repository
+into their own user account, where they can make private changes to their fork.
+To contribute these changes back into the original repository,
+a user simply creates a pull request in order to "request" that the changes be taken "upstream".
+
+Additional references:
+
+* GitHub's guide on [forking](https://guides.github.com/activities/forking/)
+* GitHub's guide on [Contributing to Open Source](https://guides.github.com/activities/contributing-to-open-source/#pull-request)
+* GitHub's guide on [Understanding the GitHub Flow](https://guides.github.com/introduction/flow/)
+
+### Building and testing
+
+To build `PSReadLine` on Windows, Linux, or macOS,
+you must have [.NET Core SDK 2.1.802 or newer](https://www.microsoft.com/net/download) installed.
+
+* Bootstrap: `./build.ps1 -Bootstrap`
+* Build:
+    * Targeting .NET 4.6.1 (Windows only): `./build.ps1 -Configuration Debug -Framework net461`
+    * Targeting .NET Core: `./build.ps1 -Configuration Debug -Framework netcoreapp2.1`
+* Test:
+    * Targeting .NET 4.6.1 (Windows only): `./build.ps1 -Test -Configuration Debug -Framework net461`
+    * Targeting .NET Core: `./build.ps1 -Test -Configuration Debug -Framework netcoreapp2.1`
+
+After build, the produced artifacts can be found at `<your-local-repo-root>/bin/Debug`.
+
+### Submitting Pull Request
+
+* If your change would fix a security vulnerability,
+  first follow the [vulnerability issue reporting policy][vuln-reporting], before submitting a PR.
+* To avoid merge conflicts, make sure your branch is rebased on the `master` branch of this repository.
+* Many code changes will require new tests,
+  so make sure you've added a new test if existing tests do not effectively cover the code changed.
+* If your change adds a new source file, ensure the appropriate copyright and license headers is on top.
+  It is standard practice to have both a copyright and license notice for each source file.
+    * For `.cs` files use the copyright header with empty line after it:
+
+    ```c#
+        // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Current guidance is to not include the last part

```suggestion
        // Copyright (c) Microsoft Corporation.
```

Reply:
Oh, OK. I copied this from PowerShell repo, so we need to clean up the files in PowerShell repo as well at some time.

Reply:
@SteveL-MSFT I also removed that part from the `Copyright` key of `PSReadLine.psd1`.
Please let me know if that one should be kept.